paddle/fluid/framework/variable.h

```diff
@@ -113,6 +123,14 @@ class Variable {
   std::shared_ptr<Placeholder> holder_;
 };
 
+inline void Variable::SharePlaceholderWith(const Variable& var) {
```

Comment:
DO NOT change in this PR because we are chasing 2.0RC1, but we should know it:

I found that variable.h doesn't split declaration and definition (i.e., .h and .cc, interface and source). In c++, it can make compilation slower, especially variable is used very common among PaddlePaddle framework.

Maybe you already know it. I mentioned it in case you don't know :-) In the future we may split it, but DO NOT change in this PR.

Reply:
Thanks,  we may split it in the future.